ui/react-components/components/FocusLock/WithFocusLock.jsx

```diff
@@ -0,0 +1,9 @@
+import FocusLock from "react-focus-lock";
```

Comment:
Can you move this to hooks folder under util.

Reply:
Done